client/src/features/visualizer-threejs/useRenderTangle.tsx

```diff
@@ -115,7 +87,7 @@ export const useRenderTangle = () => {
             const color = block.color;
 
             SPHERE_TEMP_OBJECT.position.set(x, y, z);
-            SPHERE_TEMP_OBJECT.scale.setScalar(1);
+            SPHERE_TEMP_OBJECT.scale.setScalar(INITIAL_SPHERE_SCALE);
```

Comment:
do we need to scale the sphere if we actually dont do scaling anymore?

Reply:
Yeah, the objects are created with a scale of 0, and then we set a visible scale when needed to avoid slowing down the process